inst/schema/Graph.schema.json

```diff
@@ -8,17 +8,19 @@
         "x_col": { "type": "string" },
         "y_col": { "type": "string" },
         "id_col": {"type": "string" },
-        "format": { "enum": ["long"] }
+        "format": { "enum": ["long"] },
+        "settings": { "type":  "array", "items":  {"type":  "string"}}
       },
       "additionalProperties": false,
-      "required": ["x_col","y_col","id_col","format"]
+      "required": ["x_col","id_col","format"]
```

Comment:
Should we still have any underscore names left?

Reply:
These ones are the Vue component schemas, which we should absolutely change at some point (soon).